### PR TITLE
BAEL-3943 bson to json

### PR DESCRIPTION
**Pull request by bcaure**

### CODE REVIEW

File: persistence-modules/java-mongodb/README.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+
+### Relevant articles:
+
+- [A Guide to MongoDB with Java](http://www.baeldung.com/java-mongodb)
+- [A Simple Tagging Implementation with MongoDB](http://www.baeldung.com/mongodb-tagging)
+- [MongoDB BSON Guide](https://www.baeldung.com/mongodb-bson)
+- [Geospatial Support in MongoDB](https://www.baeldung.com/mongodb-geospatial-support)
+- [Introduction to Morphia – Java ODM for MongoDB](https://www.baeldung.com/mongodb-morphia)
+- [MongoDB Aggregations Using Java](https://www.baeldung.com/java-mongodb-aggregations)
+- [MongoDB BSON to JSON](https://www.baeldung.com/bson-to-json)
```
Comment (pivovarit) [inline persistence-modules/java-mongodb/README.md:13]: Let's remove that for now since we don't want a dead link to be present there before we actually publish
```suggestion
```

File: persistence-modules/java-mongodb/README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-
-### Relevant articles:
-
-- [A Guide to MongoDB with Java](http://www.baeldung.com/java-mongodb)
-- [A Simple Tagging Implementation with MongoDB](http://www.baeldung.com/mongodb-tagging)
-- [MongoDB BSON Guide](https://www.baeldung.com/mongodb-bson)
-- [Geospatial Support in MongoDB](https://www.baeldung.com/mongodb-geospatial-support)
-- [Introduction to Morphia – Java ODM for MongoDB](https://www.baeldung.com/mongodb-morphia)
-- [MongoDB Aggregations Using Java](https://www.baeldung.com/java-mongodb-aggregations)
+## MongoDB
```
Comment (KevinGilmore) [inline persistence-modules/java-mongodb/README.md:1]: Not sure what changed here. Unless something was fixed, please revert the file.

File: persistence-modules/java-mongodb/src/main/java/com/baeldung/morphia/domain/Book.java [hunk 3]
```diff
@@ -1,116 +1,168 @@
...
-            return false;
-        if (title == null) {
-            if (other.title != null)
-                return false;
-        } else if (!title.equals(other.title))
-            return false;
-        return true;
-    }
-
+package com.baeldung.morphia.domain;
```
Comment (KevinGilmore) [inline persistence-modules/java-mongodb/src/main/java/com/baeldung/morphia/domain/Book.java:1]: Again, can't tell what was actually changed. If it's a whitespace-only change, then we should revert the changes.

File: persistence-modules/java-mongodb/README.md (hunk 2 above)
Comment (bcaure) [reply persistence-modules/java-mongodb/README.md]: Hey @KevinGilmore I fixed the bad line feeds. I'm not sure if my PR is OK so do don't hesitate to tell me what's wrong ( sorry for that...)

**Comment by bcaure** [comment]
Closing PR and opening another clean one
